shared/profile/post-proof.native.js

```diff
@@ -1,11 +1,79 @@
 // @flow
-import {Component} from 'react'
+import React from 'react'
+import {Button, CopyableText, LinkWithIcon, PlatformIcon, StandardScreen, Text} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import * as shared from './post-proof.shared'
 import type {Props} from './post-proof'
 
-class Render extends Component<void, Props, void> {
-  render () {
-    return null
+const PostProof = (props: Props) => {
+  const {
+    platform, platformUserName, descriptionText, proofAction, onCancel,
+    onComplete, isOnCompleteWaiting, errorMessage,
+  } = props
+  const {
+    descriptionView, noteText, onCompleteText, proofText, platformSubtitle, proofActionIcon, proofActionText,
+  } = shared.propsForPlatform(props)
+
+  const notification = !errorMessage ? {} : {
+    notification: {
+      type: 'error',
+      message: errorMessage,
+    },
   }
+
+  return (
+    <StandardScreen {...notification} onClose={onCancel}>
+      <PlatformIcon style={stylePlatformIcon} platform={platform} overlay='icon-proof-unfinished' overlayColor={globalColors.grey} size={48} />
+      <Text style={{...stylePlatformUsername, ...(stylePlatformSubtitle ? {} : {marginBottom: globalMargins.medium})}} type='Header'>{platformUserName}</Text>
+      {platformSubtitle && <Text style={stylePlatformSubtitle} type='Body'>{platformSubtitle}</Text>}
+      {descriptionView || (descriptionText && <Text style={styleDescriptionText} type='Body'>{descriptionText}</Text>)}
+      {proofText && <CopyableText style={styleProofContainer} value={proofText} extras$native={{style: styleProofText}} />}
```

Comment:
this should be `!!proofText` or you can get errors if you pass in an empty string `''`